panel/command/serve.py

```diff
@@ -238,7 +238,7 @@ class Serve(_BkServe):
         )),
         ('--autoreload', dict(
             action  = 'store_true',
-            help    = "Whether to autoreload source when script changes."
+            help    = "Whether to autoreload source when script changes. We recommend using --dev instead."
```

Comment:
This is correct for now, but there could be a future where `--dev` does more than `--autoreload`

Reply:
Is it just for info? Or should I remove the recommendation?